src/storage/getUnparsedAllowListsRecords.ts

```diff
@@ -3,7 +3,8 @@ import { supabase } from "@/clients/supabaseClient";
 export type UnparsedAllowListRecord = {
   claim_id: string;
   al_data_id: string;
-  data: string;
+  // TODO: Can this be typed correctly?
+  data: any;
```

Comment:
Is the data a row from the table? The it could be something like  'Tables<allowlist_records>' based on the generated types from supabase

Reply:
Nope, it's a jsonb colum. It contains the `StandardMerkleTreeData`. It's parsed as soon as it's pulled from the DB so I figured there wasnt a lot of sense spending time figuring out the types as it's basically an external dependency.

Reply:
Ah ok that should be fine then. In theory the data should be [address, uint256] or [isAddress, BigInt]. But I agree it's an external dependency